testcontainers/src/test/java/org/jdbi/v3/testing/junit5/tc/TrinoJdbiTestContainersExtensionTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.testing.junit5.tc;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.TrinoContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+class TrinoJdbiTestContainersExtensionTest extends AbstractJdbiTestcontainersExtensionTest {
+
+    @Container
+    static JdbcDatabaseContainer<?> dbContainer = new TrinoContainer("trinodb/trino");
+
+    @Override
+    JdbcDatabaseContainer<?> getDbContainer() {
+        return dbContainer;
+    }
+
+    @BeforeEach
+    public void beforeEach() throws Exception {
+        // we still love you, trino. But you are a mess.
+        Thread.sleep(5_000);
```

Comment:
Yuck.

Reply:
effect on local build times is zero. All the tests are turned off locally unless you pass `-Dbasepom.it.skip=false`. Running `make test` does that, but `mvn clean install` does not.

I don't want to spend many minutes locally to run tests either.

Reply:
the "sleep 5" is a bug workaround, when the trino container comes up, it waits for a query to succeed, but it actually calls the container "ready" when trino accepts the query and I had the test repeatedly fail with "no nodes available" because the discovery logic inside trino had not fully converged yet. The sleep ensures that the trino "cluster" is actually ready. This is a clash between the testcontainer setup and the trino engine.

Reply:
see https://github.com/testcontainers/testcontainers-java/issues/6310